build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     id("com.github.ben-manes.versions") version "0.52.0"
+    id("io.github.gradle-nexus.publish-plugin") version versions.nexus_publish_plugin
```

Comment:
`versions` are not available in root build.gradle afaik, you have to specify "2.0.0" explicitly